lib/mayaUsd/CMakeLists.txt

```diff
@@ -86,6 +86,22 @@ if (MAYA_HAS_CRASH_DETECTION)
     )
 endif()
 
+message(STATUS "MAYA_CURRENT_UFE_CAMERA_SUPPORT is ${MAYA_CURRENT_UFE_CAMERA_SUPPORT}")
```

Comment:
This section for MAYA_CURRENT_UFE_CAMERA_SUPPORT was missing from an earlier change, which I discovered while working on this.